controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -214,219 +214,6 @@ static void bml_utils_dump_conn_map(
     }
 }
 
-static void print_bml_wfca_ret_val(const std::string &func_name, int ret_val,
```

Comment:
Deleting unused code should be a separate commit. Note for next time (nitpick)

Reply:
It was used, and I replaced it...